tests/network/hotplug_bridge.go

```diff
@@ -339,10 +380,28 @@ func removeInterface(vm *v1.VirtualMachine, name string) error {
 }
 
 func verifyBridgeDynamicInterfaceChange(vmi *v1.VirtualMachineInstance, plugMethod hotplugMethod) *v1.VirtualMachineInstance {
+	virtClient := kubevirt.Client()
 	if plugMethod == migrationBased {
-		migration := libmigration.New(vmi.Name, vmi.Namespace)
-		migrationUID := libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(kubevirt.Client(), migration)
-		libmigration.ConfirmVMIPostMigration(kubevirt.Client(), vmi, migrationUID)
+		By("Waiting for MigrationRequired condition to appear")
+		Eventually(matcher.ThisVMI(vmi), 1*time.Minute, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceMigrationRequired))
+
+		By("Ensuring live-migration started")
+		var migration *v1.VirtualMachineInstanceMigration
+		Eventually(func() bool {
+			migrations, err := virtClient.VirtualMachineInstanceMigration(vmi.Namespace).List(context.Background(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, mig := range migrations.Items {
+				if mig.Spec.VMIName == vmi.Name {
+					migration = mig.DeepCopy()
```

Comment:
IMO the migration should be returned and not a boolean. Failure of a boolean says nothing in addition to its nasty look.

Reply:
A pointer to VMIM is now returned and asserted to to be nil.